generator/src/lib.rs

```diff
@@ -859,6 +859,7 @@ fn generate_function_pointers<'a>(
             }
         }
         impl #ident {
+            #[allow(clippy::cognitive_complexity)]
```

Comment:
Could we just apply this to the whole module instead? It's not like there should be any handwritten code of meaningful size here at all.

Reply:
Yeah true, I moved it to the top.